src/prommis/membrane_examples/nf/tests/test_nf_brine.py

```diff
@@ -0,0 +1,46 @@
+#################################################################################
+# adapted from test_nf.py
+#################################################################################
+
+"""
+Tests the nanofiltration flowsheet
+"""
+
+from pyomo.environ import value
+
+import pytest
+
```

Comment:
```suggestion
pytest.importorskip("watertap", reason="WaterTAP dependency not available")

```

Since WaterTAP is an optional dependency, this snippet should be added to any test file before `watertap` (or any `prommis` code that imports `watertap`) is imported.